account_invoice_report_grouped_by_picking/views/report_invoice.xml

```diff
@@ -24,31 +24,31 @@
             </tr>
         </t>
     </xpath>
-    <xpath expr="//td/span[@t-field='l.quantity']" position="before">
-        <t t-if="picking">
-            <!-- Compute product qty for that picking and invoice line -->
-            <t t-set="line_picking_qty"
-               t-value="sum(l.mapped('move_line_ids').filtered(lambda x: x.picking_id == picking).mapped('quantity_done'))"/>
-            <span t-esc="line_picking_qty"/>
-        </t>
-    </xpath>
     <xpath expr="//td/span[@t-field='l.quantity']" position="attributes">
-        <attribute name="t-if">not picking</attribute>
+        <attribute name="t-esc">lines_group['quantity']</attribute>
+        <attribute name="t-field"/>
     </xpath>
     <xpath expr="//td/span[@t-field='l.price_subtotal']" position="before">
-        <t t-if="picking">
+        <t t-if="picking" id="picking_subtotal" groups="!sale.group_show_price_total">
             <!-- Compute subtotal for that picking with discounts -->
-            <t t-set="line_picking_subtotal" t-value="line_picking_qty * (l.price_subtotal / l.quantity)"/>
+            <t t-set="line_picking_subtotal" t-value="lines_group['quantity'] * (l.price_subtotal / l.quantity)"/>
```

Comment:
Could you check, in a simple way, the sum and in the last line of the group show the difference with the accumulated?